.github/dco.yml

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
```

Comment:
here and almost everywhere else: should this remain `Power Grid Model project` or should this become `Power Grid Model IO project`?

Reply:
It is `Power Grid Model project`. This is the name of the project.`pgm-io` is the name of a library.

Reply:
yep, expected that. i will also update all copyright notices in the PGM Workshop (which has `Power Grid Model Workshop project` everywhere)